models/forms.rb

```diff
@@ -50,6 +51,67 @@ def to_ticket
 			ticket.agent_team_id = ENV['DESKPRO_TEAM_ID'].to_i if ENV['DESKPRO_TEAM_ID']
 			ticket
 		end
+
+		def to_zendesk_ticket()
+			ticket = {}
+			ticket[:subject] = subject
+			ticket[:comment] = { body: message }
+			ticket[:requester] = {
+				email: person_email,
+				name: person_name,
+			}
+			ticket[:tags] = [ 'govuk_paas_support', 'govuk_paas_product_page' ]
+			ticket[:group_id] = ENV['ZENDESK_GROUP_ID'].to_i if ENV['ZENDESK_GROUP_ID']
+			ticket
+		end
+	end
+
+	class Contact < Model
+		VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
```

Comment:
This is a really bad idea :) I would suggest to keep it as simple as possible, e.g. `/.+@.+\..+/`

Reply:
I just copied from the existing regex.

Reply:
In the forms.rb we use `:match => /.+@.+\..+/`, so should we use the same here for consistency?

Reply:
sure